File: retos-ts/14_acomodando_renos.ts
// Resultado 5 ⭐
function minMovesToStables(reindeer: number[], stables: number[]): number {
    const sortReindeer: number[] = reindeer.sort((a,b) => a - b) 
    const sortStables: number[] = stables.sort((a,b) => a - b) 
    let moves: number = 0
    
    for (let i = 0; i < reindeer.length; i++) {
        
        const tmp = sortStables[i] - sortReindeer[i]
        moves += Math.abs(tmp)
    }

    return moves
}
  

File: retos-python/14_acomodando_renos.py
# Resultado 5 ⭐
def min_moves_to_stables(reindeer, stables):
    sort_reindeer = sorted(reindeer)
    sort_stables =  sorted(stables)
    moves = 0

    for i in range(len(reindeer)):
        tmp = sort_reindeer[i] - sort_stables[i]
        moves += abs(tmp)

    return moves 